File: app/src/at-vinhnguyen/java/asiantech/internship/summer/javacore/excercise6/PrimesLessThanN.java
package asiantech.internship.summer.javacore.excercise6;

import java.util.Scanner;

public class PrimesLessThanN {

    public static void main(String[] args) {
        System.out.println("Nhap so luong so nguyen to: ");
        int n = Integer.parseInt(new Scanner(System.in).nextLine());
        System.out.println(showPrimes(n));
    }

    private static String showPrimes(int n) {
        StringBuilder builder = new StringBuilder();
        int count = 0;
        for (int j = 2; count < n; j+=2) {
            if(isPrime(j)){
                count++;
                builder.append(" ").append(j);
            }
        }
        return builder.toString();
    }

    private static boolean isPrime(int i) {
        for (int j = 2; j < i; j++) {
            if (i % j == 0) {
                return false;
            }
        }
        return true;
    }
}

File: app/src/at-vinhnguyen/java/asiantech/internship/summer/kotlincore/exercise6/PrimesLessThanN.kt
package asiantech.internship.summer.kotlincore.exercise6

import asiantech.internship.summer.kotlincore.inputIntNumber
import asiantech.internship.summer.kotlincore.isPrime

object PrimesLessThanN {

    @JvmStatic
    fun main(args: Array<String>) {
        println("Nhap so luong so nguyen to: ")
        val n = inputIntNumber()
        println(showPrimes(n))
    }

    private fun showPrimes(n: Int): String {
        val builder = StringBuilder()
        var count = 0
        var j = 2
        while (count < n) {
            if (j.isPrime()) {
                count++
                builder.append(" ").append(j)
            }
            j += 2
        }
        return builder.toString()
    }
}
